app/javascript/utils/Patient.js

```diff
@@ -59,4 +64,28 @@ function formatRace(patient) {
   return raceArray.length === 0 ? '--' : raceArray.join(', ');
 }
 
-export { formatName, formatNameAlt, formatPhoneNumber, formatRace };
+function formatDateOfBirth(data) {
+  const rowData = data.rowData;
+  if (!!rowData?.dob && moment(rowData.dob, 'YYYY-MM-DD').isAfter(moment().subtract(18, 'years'))) {
+    return (
+      <div>
+        <IconMinor patientId={rowData.id.toString()} customClass={'float-right ml-1'} />
+        {formatDate(rowData.dob)}
+      </div>
+    );
+  }
+  if (!!rowData?.date_of_birth && moment(rowData.date_of_birth, 'YYYY-MM-DD').isAfter(moment().subtract(18, 'years'))) {
```

Comment:
Let's break this into it's own util method in this file, then it can be used everywhere in the front end (specifically the Patient component